editor/src/components/canvas/canvas-strategies/strategies/convert-to-absolute-and-move-strategy.spec.browser2.tsx

```diff
@@ -1207,137 +1214,15 @@ describe('Convert to absolute/escape hatch', () => {
       )
     })
 
-    it('works on a flow element with all pins and sets hugging parent to fixed size', async () => {
```

Comment:
This test was wrong! This is not a hugging parent, so no need to set it fixed size